src/main/java/controller/logout/LogoutPresenter.java

```diff
@@ -0,0 +1,35 @@
+package controller.logout;
+
+import controller.ViewManagerModel;
+import controller.signup.SignupViewModel;
+import use_case.logout.LogoutOutputBoundary;
+import use_case.logout.LogoutOutputData;
+
+public class LogoutPresenter implements LogoutOutputBoundary {
+
+    private final ViewManagerModel viewManager;
+    private final SignupViewModel ViewModel;
+
+    public LogoutPresenter(ViewManagerModel viewManager,
+                           SignupViewModel viewModel) {
+        this.viewManager = viewManager;
+        this.ViewModel = viewModel;
+    }
+
+    @Override
+    public void prepareSuccessView(LogoutOutputData outputData) {
+        this.switchToSignupView();
```

Comment:
dont need this line, i think a better place to trigger a page switch is in HomePageView - in fact, i think prepareSuccessView and prepareFailView are both unnecessary for this case